install/rctf.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# a script to manage rCTF installations
+
+import sys, os, io
+import argparse
+import logging
+
+
+logging.basicConfig(format = '%(message)s', level = logging.DEBUG)
+
+
+# define useful functions
+
+
+def verify_privileges(euid = 0):
```

Comment:
If the user is in the docker group, they would not need root to execute `docker-compose` commands

Reply:
The installation script relies on the user being root atm and since this made to work with it, so does it. We should open an issue to enable both scripts to detect if the current user is in the `docker` group, and require root if not.

Reply:
Made it: https://github.com/redpwn/rctf/issues/43